tests/test_config_parsing/test_site_config.py

```diff
@@ -38,3 +38,13 @@ def test_site_config_defaults_umask(tmpdir):
             "RUNPATH_FILE": "rpfile",
         }
     )
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@given(config_dicts())
+def test_site_config_dict_without_umask_works(config_dict):
+    config_dict.pop(ConfigKeys.UMASK)
+    try:
+        SiteConfig(config_dict=config_dict)
```

Comment:
should this test also verify the default value is set  and used?

Reply:
that's a good question. I think it might make sense. Gonna add that.

Reply:
it is done.